cypress/fixtures/list_of_articles.json

```diff
@@ -1,22 +1,30 @@
-{
-  "data": [
-    {
-      "id": 1,
-      "title": "A Day in Stockholm",
-      "ingress": "How to spend 24 hours in Stockholm",
-      "image": "https://image.freepik.com/free-photo/sailing-boats-yachts-pier-stockholm-front-city-center_72229-307.jpg"
-    },
-    {
-      "id": 2,
-      "title": "A Day in Paris",
-      "ingress": "How to spend 24 hours in Paris",
-      "image": "https://image.freepik.com/free-photo/paris-city-view-from-high_1160-763.jpg"
-    },
-    {
-      "id": 3,
-      "title": "A Day in Rome",
-      "ingress": "How to spend 24 hours in Rome",
-      "image": "https://image.freepik.com/free-photo/roman-coliseum-seen-from-afar_1219-15.jpg"
-    }
-  ]
-}
+[
+  {
+    "id": 36,
+    "title": "A Day in Stockholm",
+    "ingress": "How to spend 24 hours in Stockholm",
+    "image": "https://image.freepik.com/free-photo/sailing-boats-yachts-pier-stockholm-front-city-center_72229-307.jpg"
+  },
+  {
+    "id": 37,
+    "title": "A Day in Paris",
+    "ingress": "How to spend 24 hours in Paris",
+    "image": "https://image.freepik.com/free-photo/paris-city-view-from-high_1160-763.jpg"
+  },
+  {
+    "id": 38,
+    "title": "A Day in Rome",
+    "ingress": "How to spend 24 hours in Rome",
+    "image": "https://image.freepik.com/free-photo/roman-coliseum-seen-from-afar_1219-15.jpg"
+  },
+  {
+    "id": 9,
```

Comment:
Nice! Just out of interest, How come you changed the id's and did not add the category_id to all of them? :)

Reply:
These are the mock data files. Category id is not required.
We changed the id cause we were testing both with mock data and the real back-end to make sure that everything is functional. And everything is. Both with mock data and in live env.